contracts/finality/src/exec/finality.rs

```diff
@@ -406,11 +415,13 @@ pub(crate) mod tests {
         let add_finality_signature = get_add_finality_sig();
         let proof = add_finality_signature.proof.unwrap();
 
+        let current_epoch = 1;
+
         // Convert the PubRandCommit in the type defined in this contract
         let pr_commit = PubRandCommit {
             start_height: pr_commit.start_height,
             num_pub_rand: pr_commit.num_pub_rand,
-            height: pr_commit.height,
+            epoch: current_epoch,
             commitment: pr_commit.commitment,
```

Comment:
Hmm wondering whether this test case can pass given that we did not mock the last finalised epoch is no smaller than `current_epoch`